### PR TITLE
autoflake: add page

### LINKED ISSUE #1662: page request: autoflake
Webpage here - https://github.com/myint/autoflake#example

Please go through the [CONTRIBUTING](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) page to understand what is the format we use for our pages.

### PR DESCRIPTION
**Pull request by andrewda**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines 

Closes #1662

### CODE REVIEW

File: pages/common/autoflake.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# autoflake
+
+> A tool to remove unused imports and variables from Python code.
+
+- Remove unused variables from a single file and display diff:
```
Comment (sbrl) [inline pages/common/autoflake.md]: Perhaps ...`a diff` would read better here?

Comment (andrewda) [reply pages/common/autoflake.md]: How about `the diff`?

Comment (sbrl) [reply pages/common/autoflake.md]: Sure!

Comment (andrewda) [reply pages/common/autoflake.md]: 👍  Updated

File: pages/common/autoflake.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+`autoflake --remove-all-unused-imports {{file1.py}} {{file2.py}} {{file3.py}}`
+
+- Remove unused variables from a file and make changes to that file:
+
+`autoflake --remove-unused-variables --in-place {{file.py}}`
+
+- Remove unused variables recursively from all files in a directory:
+
+`autoflake --remove-unused-variables --recursive {{/path/to/directory}}`
```
Comment (agnivade) [inline pages/common/autoflake.md]: Remove preceding slash from "/path"

File: pages/common/autoflake.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- Remove unused imports from multiple files and display the diffs:
+
+`autoflake --remove-all-unused-imports {{file1.py}} {{file2.py}} {{file3.py}}`
+
+- Remove unused variables from a file and make changes to that file:
+
+`autoflake --remove-unused-variables --in-place {{file.py}}`
+
+- Remove unused variables recursively from all files in a directory:
```
Comment (agnivade) [inline pages/common/autoflake.md]: Will this display the diff or change the file in-place ? I think we should add the `--in-place` flag to make it clear.

File: pages/common/autoflake.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+- Remove unused variables from a single file and display the diff:
+
+`autoflake --remove-unused-variables {{file.py}}`
+
+- Remove unused imports from multiple files and display the diffs:
+
+`autoflake --remove-all-unused-imports {{file1.py}} {{file2.py}} {{file3.py}}`
+
+- Remove unused variables from a file and make changes to that file:
```
Comment (agnivade) [inline pages/common/autoflake.md]: Change this to "Remove unused variables from a file, overwriting the file".

**Comment by sbrl** [comment]
Hey! Thanks for the changes in response to @agnivade's comments. In future, could you simply commit to your branch again please, instead of squashing? As maintainers we get a special squash button that we can use when merging your changes, and it helps us see what you've changed. Thanks! 😺

**Comment by sbrl** [comment]
Looks ok to me! Thanks, @andrewda :D

**Pull request merged**